rest/blip_sync.go

```diff
@@ -665,17 +706,33 @@ func isCompressible(filename string, meta map[string]interface{}) bool {
 	return true // be optimistic by default
 }
 
+func (bh *blipHandler) logEndpointEntry(profile string, endpoint fmt.Stringer) {
+	bh.LogTo("SyncMsg", "#%03d: Prof:%s %s User:%s", bh.serialNumber, profile, endpoint, bh.effectiveUsername)
+}
```

Comment:
What's the benefit in passing endpoint as a stringer, instead of just a string?  Given the choice between:

```
bh.logEndpointEntry(rq.Profile(), getAttachment)
bh.logEndpointEntry(rq.Profile(), NewAdhocStringer(fmt.Sprintf("#Changes: %d", len(changeList))))
```
versus
```
bh.logEndpointEntry(rq.Profile(), getAttachment.String())
bh.logEndpointEntry(rq.Profile(), fmt.Sprintf("#Changes: %d", len(changeList)))
```
...I think the latter is much easier to read/write.

Reply:
I think I had something in mind originally that never ended up being useful, so I'll switch this back to just plain strings.